NEWS.md

```diff
@@ -478,6 +478,17 @@ Deprecated or removed
   * The no-op `transpose` fallback has been deprecated. Consider introducing suitable
     `transpose` methods or calling `permutedims(x, [2,1])` ([#13171], [#17075], [#17374]).
 
+  * The following macros have been deprecated ([#6674] and [#4233]):
```

Comment:
Should it be the PR number of this PR, or the one where the macros were actually deprecated? Sorry, this is my first ever PR to anything open source, still learning my way around a bit!